src/bucket.js

```diff
@@ -69,23 +69,24 @@ export default class Bucket {
    * @param  {Boolean}  options.safe  The safe option.
    * @return {Collection}
    */
-  collection(name, options) {
+  collection(name, options={}) {
     return new Collection(this.client, this, name, this._bucketOptions(options));
   }
 
 
   /**
-   * Retrieves bucket properties.
+   * Retrieves bucket data.
    *
    * @param  {Object} options         The options object.
    * @param  {Object} options.headers The headers object option.
    * @return {Promise<Object, Error>}
    */
-  getAttributes(options={}) {
+  getData(options={}) {
     return this.client.execute({
       path: endpoint("bucket", this.name),
       headers: {...this.options.headers, ...options.headers}
-    });
+    })
+    .then((res) => res.data);
```

Comment:
This is now the only method having this return value signature, I suspect this could be confusing to users; thoughts?